lib/datasource/index.ts

```diff
@@ -106,6 +108,7 @@ async function fetchReleases(
   }
   const dep = await datasources[datasource].getPkgReleases(config);
   addMetaData(dep, datasource, config.lookupName);
+  dep.sourceUrl = sanitizeSourceUrl(dep.sourceUrl);
```

Comment:
This behaviour will result in every result without a sourceUrl now having one that is `null`. It's cleaner if it's simply absent than null. i.e. `sanitizeSourceUrl() === null`, which is not desirable.

Reply:
https://github.com/ocadotechnology/renovate/commit/76559ef421d5d3b69ba6c98cafa0741589456e0d#diff-a367c71b38b3046d1795a93024b87f28
should fix this.